src/legacy/config/config-set.spec.ts

```diff
@@ -687,10 +688,10 @@ describe('resolvePath', () => {
 }) // resolvePath
 
 describe('_resolveTsConfig', () => {
-  let findConfig!: jest.SpyInstance<string | undefined>
```

Comment:
Second thought. It should be possible to replace everything like this:

```diff
-  let findConfig!: jest.SpyInstance<string | undefined>
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  let readConfig!: jest.SpyInstance<{ config?: any; error?: ts.Diagnostic }>
-  let parseConfig!: jest.SpyInstance<ts.ParsedCommandLine>
-  let cs!: ConfigSet
-
-  beforeAll(() => {
-    findConfig = jest.spyOn(ts, 'findConfigFile')
-    readConfig = jest.spyOn(ts, 'readConfigFile')
-    parseConfig = jest.spyOn(ts, 'parseJsonConfigFileContent')
-  })

+  const findConfig = jest.spyOn(ts, 'findConfigFile')
+  const readConfig = jest.spyOn(ts, 'readConfigFile')
+  const parseConfig = jest.spyOn(ts, 'parseJsonConfigFileContent')
+  let cs!: ConfigSet
```

State and implementation of mocks is reset / replaced between tests. Seems like it works, but please do double check (;

Reply:
Looks like it works indeed.